microcosm_logging/factories.py

```diff
@@ -10,6 +10,7 @@
 
 @defaults(
     default_format="%(asctime)s - %(name)-12s - [%(levelname)s] - %(message)s",
+    format_string="{asctime} - {name} - [{levelname}] - {message}",
```

Comment:
Good question. I'm using `default_formatter` to configure in old-style `logging.Formatter` as the default formatter. I don't know if python uses the default formatter for some `default` handler in addition to the console handler, but I thought it made sense to have a default that uses the old style in case things expect a default formatter instead of a more specific handler.